sdks/python/apache_beam/utils/annotations.py

```diff
@@ -90,6 +90,8 @@ def inner(*args, **kwargs):
       message = '%s is %s' % (fnc.__name__, label)
```

Comment:
I take you mean then ignore the other arguments (since, current). I am trying to keep the original structure.